abstract suggested (#49)
SpecialeBajs/speciale-rapport: article/Abstract.tex

Graph Convolutional Networks (GCN) is a State-of-the-Art method used for recommendation.
Throughout this paper we suggest two new alternative extensions for LightGCN called Aggressive Layer Combination (ALC) and Balanced Layer Combination (BLC).

> These are used as alternative layer combination methods for graph convolutional networks instead of using means.

Reviewer: Er enig i at der burde stå means men lgcn kalder det weighted summation og det kalder vi den også i resten af rapporten så måske vi skulle ændre det til det?

This showed large improvements on datasets where items have few connections and users have a larger amount of connections.

> We also conduct an ablation study for GCF to understand why it outperformed LightGCN on their datasets.

Reviewer 2: ```suggestion
An ablation study for GCF is also conducted to understand why it outperformed LightGCN on their datasets.
```

> Both ALC and BLC showed to outperform GCF.

Reviewer 2: ```suggestion
ALC and BLC both outperformed GCF.
```

> Additionally we experimented with some edge cases, where only one embedding layer were used and only removing the 0th embedding layer.

Reviewer 2: ```suggestion
Additionally, we experimented with some edge cases, where only one embedding layer was used or where the 0th embedding is removed.
```
